File: src/main/java/com/drunkensoftball/api/auth/repo/AuthenticationRepository.java
package com.drunkensoftball.api.auth.repo;

import com.drunkensoftball.api.auth.domain.Authentication;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AuthenticationRepository extends JpaRepository<Authentication, Long> {

    Authentication findByToken(String token);

}

File: src/main/kotlin/com/drunkensoftball/api/auth/repo/AuthenticationRepository.kt
package com.drunkensoftball.api.auth.repo

import com.drunkensoftball.api.auth.domain.AuthenticationEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository


@Repository
interface AuthenticationRepository : JpaRepository<AuthenticationEntity, Long> {
    fun findByToken(token: String): AuthenticationEntity?
}
